tests/NovaTestTraitTest.php

```diff
@@ -34,12 +34,12 @@ public function testGetNovaResourceUri()
     {
         $result = $this->generateNovaUri(TestModel::class);
 
-        $this->assertEquals($result, '/nova-api/test-model-resources');
+        $this->assertEquals($result, '/nova-api/test-models');
```

Comment:
please implement test case for the `Media` class to check that pluralizing works